proc_CPMG_mw.py

```diff
@@ -532,20 +523,15 @@ def print_fun(x, f, accepted):
 
 for i,k in enumerate(power_list):
     temp = data['power',i].C
-    x = tE_axis 
-    ydata = temp.data.real
-    fl.next('DECAY DATA')
-    fl.plot(x,ydata, '.', alpha=0.4, label='%f'%k, human_units=False)
-    fitfunc = lambda p, x: p[0]*exp(-x/p[1])
-    errfunc = lambda p_arg, x_arg, y_arg: fitfunc(p_arg, x_arg) - y_arg
-    p0 = [11102.0,0.6]
-    p1, success = leastsq(errfunc, p0[:], args=(x, ydata))
-    x_fit = linspace(x.min(),x.max(),20000)
+    fl.plot(temp, '.', alpha=0.4, label='%f'%k, human_units=False)
+    temp = fitdata(temp)
+    A,R2,tau = symbols('A R2 tau')
+    data.symbolic_expression = A*sympy.exp(-tau*R2)
+    data.guess({A:11102.0,R2:0.6})
+    data.fit()
```

Comment:
can this be replaced with the new "decay" function?

Reply:
possibly? I do not know what specifically this processing script is used for? So I don't really know it's purpose

Reply:
I would either try or create an issue to follow up

Reply:
please note here which you end  up doing

Reply:
according to alec this processes a DNP  with CPMG detection

Reply:
"which you end up doing" := did you try to replace with the `decay` function here, or did you create an issue, and if you created an issue, please reference by number (I would imagine it's faster to just replace with the `decay` function, but unanticipated errors mean that you should put off to an issue)

Reply:
Issue #25